Cargo.toml

```diff
@@ -792,6 +792,7 @@ environmental = { version = "1.1.4", default-features = false }
 equivocation-detector = { path = "bridges/relays/equivocation" }
 ethabi = { version = "2.0.0", default-features = false, package = "ethabi-decode" }
 ethbloom = { version = "0.14.1", default-features = false }
+ethereum = { version = "0.18.2", default-features = false }
```

Comment:
Do we need this? If yes we should use a modern crate: https://docs.rs/alloy-consensus/latest/alloy_consensus/